includes/definitions.inc.php

```diff
@@ -1435,6 +1435,19 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+// Lenovo EMC (NAS)
+$os = 'lenovoemc';
```

Comment:
BTW, this is the box (just in case it helps),
http://shop.lenovo.com/us/en/servers/network-storage/lenovoemc/ix2/